src/Directory.Version.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>4.$(MinorVersionPrefix)39.100</VersionPrefix>
+    <VersionPrefix>4.1039.200</VersionPrefix>
```

Comment:
This looks like a mistake

Reply:
Yes, this happened because I am running the pipeline from the main branch instead of in-proc. I have to cherry-pick a fix from main to in-proc to be able to run from in-proc. I will follow up with that PR later today. In the meantime, I will rerun the pipeline and replate this PR.